velox/docs/functions/spark/bitwise.rst

```diff
@@ -12,6 +12,17 @@ Bitwise Functions
     Returns the bitwise OR of ``x`` and ``y`` in 2's complement representation.
     Corresponds to Spark's operator ``^``.
 
+.. spark:function:: bit_count(x) -> integer
+
+    Returns the number of bits that are set in the argument ``x`` as an unsigned 64-bit integer,
+    or NULL if the argument is NULL.
+
+.. spark:function:: bit_get(x, pos) -> byte
```

Comment:
byte is not a supported type; I assume this should be tinyint

What is the return type of bit_get in Spark?

Reply:
bit_get return ByteType in Spark, it's should equals to tinyint in Velox, thanks!
https://github.com/apache/spark/blob/94031ead78682bd5c1adab8b87e61055968c8998/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/bitwiseExpressions.scala#L273-L278